OpenOversight/app/templates/about.html

```diff
@@ -20,19 +20,9 @@
   <div class="vertical-padding">
     <p class="lead">
       <small>
-        OpenOversight is a
-        <a
-          href="https://lucyparsonslabs.com"
-          rel="noopener noreferrer"
-          target="_blank"
-          >Lucy Parsons Labs</a
-        >
-        project that aims to improve law enforcement visibility and transparency
-        using public and crowdsourced data. We maintain databases, digital
-        galleries, and profiles of individual law enforcement officers from
-        departments across the United States that consolidate information
-        including names, birthdates, mentions in news articles, salaries, and
-        photographs.
+        OpenOversight is a <a href="" target="_blank" rel="noopener noreferrer">Seattle Tech Bloc</a> project that aims to improve law enforcement
```

Comment:
Was this supposed to link to our twitter?

Reply:
Whoops yes!